crates/provider-sdk/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "provider-sdk"
```

Comment:
This name is way too generic, please use `wasmcloud-provider`, as outlined in https://github.com/wasmCloud/wasmCloud/issues/387 (also the directory should be renamed)

Reply:
I think the `sdk` is important. Will preface `wasmcloud` though